app/components/Layout/StaticLayout.tsx

```diff
@@ -18,6 +20,7 @@ const StaticLayout: React.FC<Props> = ({
   return (
```

Comment:
`StaticLayout` is used on the 400/500 and fallback pages, and it looks like the subheader is hidden, but the footer isn't. We could maybe do some conditional logic to show the appropriate footer based on where the user came from, but that seems complicated for a first slice. Maybe we can hide the footer in static layout, or talk to the designers about if the internal/external footers have to be different.

Reply:
Opened a new issue for this discussion: https://github.com/bcgov/cas-cif/issues/1456